ios/application_swiftui/Resources/Info.plist

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleExecutable</key>
+	<string>@@PROJECTNAME@@</string>
+	<key>CFBundleIcons</key>
+	<dict>
+		<key>CFBundlePrimaryIcon</key>
+		<dict>
+			<key>CFBundleIconFiles</key>
+			<array>
+				<string>AppIcon29x29</string>
+				<string>AppIcon40x40</string>
+				<string>AppIcon57x57</string>
+				<string>AppIcon60x60</string>
+			</array>
+			<key>UIPrerenderedIcon</key>
+			<true/>
+		</dict>
+	</dict>
+	<key>CFBundleIcons~ipad</key>
+	<dict>
+		<key>CFBundlePrimaryIcon</key>
+		<dict>
+			<key>CFBundleIconFiles</key>
+			<array>
+				<string>AppIcon29x29</string>
+				<string>AppIcon40x40</string>
+				<string>AppIcon57x57</string>
+				<string>AppIcon60x60</string>
+				<string>AppIcon50x50</string>
+				<string>AppIcon72x72</string>
+				<string>AppIcon76x76</string>
+			</array>
+			<key>UIPrerenderedIcon</key>
+			<true/>
+		</dict>
+	</dict>
+	<key>CFBundleIdentifier</key>
+	<string>@@PACKAGENAME@@</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>CFBundleSignature</key>
+	<string>????</string>
+	<key>CFBundleSupportedPlatforms</key>
+	<array>
+		<string>iPhoneOS</string>
+	</array>
+	<key>CFBundleVersion</key>
+	<string>1.0</string>
+	<key>LSRequiresIPhoneOS</key>
+	<true/>
+	<key>UIDeviceFamily</key>
+	<array>
+		<integer>1</integer>
+		<integer>2</integer>
+	</array>
+	<key>UIRequiredDeviceCapabilities</key>
+	<array>
+		<string>armv7</string>
+	</array>
+	<key>UILaunchImageFile</key>
+	<string>LaunchImage</string>
+	<key>UILaunchImages</key>
```

Comment:
Since you're targeting 14+ anyway, can we use [UILaunchScreen](https://developer.apple.com/documentation/bundleresources/information_property_list/uilaunchscreen) instead and get rid of all of the launch images?

Reply:
I got the relevant xml, but can't get to show the launch image on launch for some reason. This is how the [info.plist](https://www.toptal.com/developers/hastebin/itukayenag.xml) looks right now for the test project, any ideas? :/ Testing with [this image](https://i.ibb.co/vVDb64D/Launch-Image.jpg). I tried both with and without the file name extension.

Reply:
@kabiroberai I got this xml now from a fresh SwiftUI xcode project, so to keep only the relevant stuff, but it still doesn't show. The image shows correctly on Filza, so not sure what else I could do. Thoughts? I can't also compare properly since I don't really have access to a Mac.

```xml
<key>UILaunchScreen</key>
	<dict>
		<key>UIImageName</key>
		<string>LaunchImage</string>
	</dict>```

Reply:
I don't believe these have ever worked when built on non-Darwin hosts. For my own project, I leave it as 
```xml
<key>UILaunchStoryboardName</key>
<string>LaunchScreen</string>
```
which seems to adjust the size accordingly and simultaneously removes the need for stub images (courtesy of dlevi)

Reply:
Yeah, just got rid of the placeholder images in favor of this, thanks

Reply:
I still want to try and figure this out if possible

> I don't believe these have ever worked when built on non-Darwin hosts

@L1ghtmann is this because the LaunchImage is expected to be in the asset catalog? @Luki120 did you add 2x and 3x sizes for the image?

Reply:
Also, if we can't get an image to work without loading it into the asset catalog, what about using an empty dict? The docs seem to indicate that doing so should result in a `systemBackground` fill.

Reply:
> is this because the LaunchImage is expected to be in the asset catalog? 

Not sure. I've only ever developed apps following the format of Theos's app template (don't have access to Xcode). Tried in the past to mimic the LaunchImage impl in Xcode's app template, but none of my attempts ever proved successful.

Reply:
Yeah I think the issue is the lack of asset catalog support. Let me know if an empty UILaunchScreen dict works.

Reply:
Just pushed https://github.com/Luki120/templates/commit/9f266faea4c87648bcc89652e7250c1433bff4f3, I sent two videos to #theos-general comparing with & without the empty dict and it does in fact result in a `systemBackground` fill, pinged you there silently